ibm_catalog.json

```diff
@@ -47,10 +47,385 @@
       ],
       "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-code-engine/issues](https://github.com/terraform-ibm-modules/terraform-ibm-code-engine/issues). Please note this product is not supported via the IBM Cloud Support Center.",
       "flavors": [
+        {
+          "label": "[Experimental] Standard - Integrated setup with configurable services",
```

Comment:
Remove `[Experimental]` - this is a community DA - we don't need to call this experimental (thats only for public catalog DAs using addons)

Reply:
done